### ISSUE TITLE
.docx: incorrect rendering of tables

### ISSUE THREAD

**Issue by parrenin** [ONLYOFFICE/DocumentServer#1506]
### This issue is unique.

- [X] I have used the [search tool](https://github.com/ONLYOFFICE/DesktopEditors/issues?q=) and did not find an issue describing my bug.

### Operating System

Linux (DEB package)

### Version information

6.4.1.46.

### Expected Behavior

When opening the attached file in Word and OO, the rendering should be the same.
[fiche objectifs tectonique.docx](https://github.com/ONLYOFFICE/DesktopEditors/files/7499384/fiche.objectifs.tectonique.docx)

### Actual Behavior

The file renders differently in Word and OO.
In OO, it seems the two tables are super-imposed.

### Reproduction Steps

1. Open the attached .docx file in Word
2. Open it in OO
3. Compare

**Comment by parrenin** [comment]
Still present in OO 7.0.0

**Comment by ShockwaveNN** [comment]
Yes, we will cose issue as soon as our dev team is fixed it, but it's not fixed

**Comment by parrenin** [comment]
Still repro in 7.0.2.
